config/routes.rb

```diff
@@ -73,6 +73,8 @@
     get "/paypal", to: "paypal#express", as: :paypal_express
     get "/paypal/confirm", to: "paypal#confirm", as: :confirm_paypal
     get "/paypal/cancel", to: "paypal#cancel", as: :cancel_paypal
+
+    get "/stripe/confirm", to: "stripe#confirm", as: :confirm_stripe
```

Comment:
Yay! :tada: 

The ressourcful Rails way would have been `StripeConfirmation#create` but you probably need a GET request, right? `StripeConfirmation#show` could have worked but is a bit of a hack as well.

Reply:
Yeah, I was quite conflicted here! Part of the consideration was around attempting to bring it in line with the Paypal controller, and _that_ has a couple of fixed named routes which I think we can't change (express, confirm, cancel).